src/ngraph/runtime/cpu/cpu_external_function.cpp

```diff
@@ -23,6 +23,8 @@
 #include <typeinfo>
 #include <unordered_map>
 
+#define __TBB_PREVIEW_LIGHTWEIGHT_POLICY 1
```

Comment:
you will probably have the same style issue on CI as last time. You might want to temporarily disable some checks around this header

Reply:
Last time I saw warnings when I built it before moving the header to the top. Now I do not see warnings when I build it. Will CI build have warnings?